source/com.android.billingclient/billing/Additions/Additions.cs

```diff
@@ -151,11 +117,7 @@ public Task<QueryProductDetailsResult> QueryProductDetailsAsync(QueryProductDeta
 
             var listener = new InternalProductDetailsResponseListener
             {
-                ProductDetailsResponseHandler = (r, s) => tcs.TrySetResult(new QueryProductDetailsResult
-                {
-                    Result = r,
-                    ProductDetails = s
-                })
+                ProductDetailsResponseHandler = (r, queryResult) => tcs.TrySetResult(queryResult)
```

Comment:
@mattleibow why is the billing result `r` no longer returned here - only `queryResult`?

Reply:
This API isn't marked `[Obsolete]` so we just missed this in code review. Seeing if we can restore it here:
* https://github.com/dotnet/android-libraries/issues/1206

Reply:
If you have some time to test:
* https://github.com/dotnet/android-libraries/pull/1207#issuecomment-3080283074